pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -139,25 +140,41 @@ private static URI getURI(String protocol, String host, int port, String path)
     return new URI(protocol, null, host, port, path, null, null);
   }
 
+  /**
+   * Deprecated due to lack of protocol/scheme support. May break for deployments with TLS/SSL enabled
```

Comment:
Can you provide a pointer to the method to be used instead? Do we need to introduce a method that takes authToken as an additional argument ? Perhaps it should specify "null" (the string) in the calling sequence?

Reply:
added